hack/build.sh

```diff
@@ -1,19 +1,24 @@
 #!/bin/bash -e
 # This script is used to build, test and squash the OpenShift Docker images.
 #
-# $1 - Specifies distribution - "rhel7" or "centos7"
-# $2 - Specifies the image version - (must match with subdirectory in repo)
+# OS - Specifies distribution - "rhel7" or "centos7"
+# VERSION - Specifies the image version - (must match with subdirectory in repo)
```

Comment:
you should add BASE_IMAGE and OPENSHIFT_NAMESPACES to the list of variables this script consumes.